### ISSUE TITLE
Add I18N to generated report

### ISSUE THREAD

**Issue by alani1** [alani1/KALanguageReport#6]
The reports generated should be in the target language e.g. Chinese in Chinese.
Follow https://docs.djangoproject.com/en/1.7/topics/i18n/ to add I18N and translate to Chinese,German and Korean.

**Comment by KevinHu2014** [comment]
I should study this http://www.djangobook.com/en/2.0/chapter19.html     right?

**Comment by alani1** [comment]
Yes, this tutorial will give you some ideas how I18 should be done in the
django framework. Then we will have to add this to the templates and code
to generate the reports in the target language.
